scripts/staking.py

```diff
@@ -82,6 +124,9 @@
         owner_crypto = EthereumCrypto(
             private_key_path=args.owner_private_key_path, password=args.password
         )
+
+        unstake_everest(ledger_api, args.service_id, owner_crypto)
```

Comment:
@mariapiamo please review this.

Before ANY staking operation takes place, the script verifies if the service is staked on Everest, and requires to unstake from Everest before continuing.

Reply:
Ok, but check my comment above the unstake_everest